docs/release-notes.rst

```diff
@@ -99,12 +125,12 @@ Version 0.26.5
 
 **New Features**
 
--  Authentication: Users can now provide a Pachyderm address in the master config under
-   ``integrations.pachyderm.address``. This address will be added as an environment variable called
-   ``PACHD_ADDRESS`` in task containers. The OIDC raw ID token will also be available as an
-   environment variable called ``DEX_TOKEN`` in task containers.
+-  Authentication: *(Enterprise edition only)* Users can now provide a Pachyderm address in the
+   master config under ``integrations.pachyderm.address``. This address will be added as an
+   environment variable called ``PACHD_ADDRESS`` in task containers. The OIDC raw ID token will also
+   be available as an environment variable called ``DEX_TOKEN`` in task containers.
 
--  Authentication: In the enterprise edition, add synchronization of OIDC user group memberships
+-  Authentication: *(Enterprise edition only)* Add synchronization of OIDC user group memberships
    with existing groups. Configure by setting ``oidc.groups_attribute_name`` in the master config to
    the string value of the authenticator's claim name for groups.
```

Comment:
in 0.26.5 wasn't it set as `oidc.groups_claim_name` and then it switched to `oidc.groups_attribute_name` later (perhaps in 0.26.7?)

Reply:
Fixed!